rust/connlib/clients/shared/src/eventloop.rs

```diff
@@ -140,6 +143,9 @@ where
                         .send(PHOENIX_TOPIC, EgressMessages::ReuseConnection(connection));
                 }
             }
+            firezone_tunnel::ClientEvent::RefreshInterfance => {
+                unreachable!("Handled internally");
+            }
```

Comment:
I'd like to not re-introduce something like this now that we have removed the old instances of it. We should make an internal event or something for the state to communicate with the `Io` runtime.